my_game.py

```diff
@@ -99,12 +99,29 @@ def on_show_view(self):
             scale=SCALING,
         )
 
+        # create a sample enemy
+        self.sample_enemy = Enemy(
+            filename="images/tiny_dungeon/Tiles/tile_0087.png",
+            center_pos=(200, 200),
+            max_hp=10,
+            speed=1,
+            impassables=self.tilemap.sprite_lists["impassable"],
+            grid_size=int(self.tilemap.tile_width),
+            boundary_left=0,
+            boundary_right=SCREEN_WIDTH,
+            boundary_bottom=0,
+            boundary_top=SCREEN_HEIGHT,
+            scale=SCALING
+        )
+        
         # Register player and walls with physics engine
         self.physics_engine =  arcade.PhysicsEngineSimple(
             player_sprite=self.player,
             walls = self.tilemap.sprite_lists["impassable"]
         )
 
+        self.sample_enemy.go_to_point((100, 100))
```

Comment:
Change name _go_to_position()_ to make it clear that the argument is a _position_?